common/lib/xmodule/xmodule/modulestore/tests/test_cross_modulestore_import_export.py

```diff
@@ -448,3 +450,73 @@ def test_round_trip(self, source_builder, dest_builder, source_content_builder,
                             dest_store,
                             dest_course_key,
                         )
+
+    @ddt.data(*itertools.product(
+        SPLIT_MODULESTORE_SETUP,
+        SPLIT_MODULESTORE_SETUP,
+        CONTENTSTORE_SETUPS,
+        CONTENTSTORE_SETUPS,
+        ['split_course_with_static_tabs'],
+    ))
+    @ddt.unpack
+    def test_split_course_export_import(
+            self,
+            source_builder,
+            dest_builder,
+            source_content_builder,
+            dest_content_builder,
+            course_data_name
+    ):
+        # Construct the contentstore for storing the first import
+        with source_content_builder.build() as source_content:
+            # Construct the modulestore for storing the first import (using the previously created contentstore)
+            with source_builder.build(contentstore=source_content) as source_store:
+                # Construct the contentstore for storing the second import
+                with dest_content_builder.build() as dest_content:
+                    # Construct the modulestore for storing the second import (using the second contentstore)
+                    with dest_builder.build(contentstore=dest_content) as dest_store:
+                        source_course_key = source_store.make_course_key('a', 'source', '2015_Fall')
+                        dest_course_key = dest_store.make_course_key('a', 'dest', '2015_Fall')
+
+                        import_course_from_xml(
+                            source_store,
+                            'test_user',
+                            TEST_DATA_DIR,
+                            source_dirs=[course_data_name],
+                            static_content_store=source_content,
+                            target_id=source_course_key,
+                            raise_on_failure=True,
+                            create_if_not_present=True,
+                        )
+
+                        export_course_to_xml(
+                            source_store,
+                            source_content,
+                            source_course_key,
+                            self.export_dir,
+                            'exported_source_course',
```

Comment:
Make this a definition and use the def in the multiple places that use the string currently.